server/src/main/java/com/paypal/selion/grid/FileExtractor.java

```diff
@@ -81,6 +81,7 @@ static String getFileNameFromPath(String name) {
         LOGGER.entering(archiveFile);
 
         LOGGER.info("Extracting " + archiveFile);
+        String tempArchiveFile = archiveFile;
```

Comment:
Similar feedback. Let's do the assignment to a new variable when/where the assignment happens. Let's keep the 'reads' / conditionals in this method acting on the method parameter `archiveFile`